src/app/games/lasca/lasca.component.html

```diff
@@ -0,0 +1,48 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+     width="100%" height="auto"
+     [attr.viewBox]="LEFT + ' ' + UP + ' ' + WIDTH + ' ' + HEIGHT"
+     preserveAspectRatio="xMidYMid meet">
+    <defs>
+        <g id="piece">
+            <path d="M 10 15 v 15 a 40 15, 0, 0, 0, 80 0 v -15 a 40 15, 0, 0, 0, -80 0 Z"/>
+            <ellipse cx="50" cy="15" rx="40" ry="15"/>
+        </g>
+    </defs>
+    <g *ngFor="let line of adaptedBoard; index as y">
+        <g *ngFor="let spaceInfo of line; index as x">
+            <g id="space_{{ x }}_{{ y }}"
+               (click)="onClick(x, y)"
+               [attr.transform]="getCoordTransform(x, y)">
+                <polygon id="rhombus_{{ x }}_{{ y }}"
+                         [attr.points]="getRhombusPoints()"
+                         [ngClass]="spaceInfo.spaceClasses"
+                         class="base-no-fill"
+                         [attr.fill]="backgroundColor(x, y)"/>
+                <g *ngFor="let pieceInfo of spaceInfo.pieceInfos; index as z"
+                    pointer-events="fill"
+                    [attr.transform]="'translate(15 ' + (35 - (z * 15)) + ')'"
+                    class="base mid-small-stroke">
+                    <use id="space_{{ x }}_{{ y }}_piece_{{ z }}"
+                         x="10"
+                         y="0"
+                         xlink:href="#piece"
+                         [ngClass]="pieceInfo.classes"
+                         class="base mid-stroke"/>
+                    <circle id="space_{{ x }}_{{ y }}_piece_{{ z }}_officer_symbol"
+                            *ngIf="pieceInfo.isOfficer"
+                            [attr.cx]="10 + (SPACE_SIZE * 0.5)"
+                            [attr.cy]="37.5"
+                            [attr.r]="6"
+                            [ngClass]="pieceInfo.classes"/>
+                </g>
+            </g>
+        </g>
+    </g>
+    <rect x="0"
+          y="700"
+          [attr.width]="700 * HORIZONTAL_WIDTH_RATIO"
+          [attr.height]="EPAISSEUR"
+          class="base"/>
+    <polygon [attr.points]="getDiagonalVolumeLanPoints()"
```

Comment:
LAN Points ? :o

Reply:
DIALOGAN !
C't'une typo :p
Sinon "diagonal edge point" plutôt ?

Reply:
c'est quoi diagonal edge point ? Ça représente quoi exactement ?

Reply:
je voulais dire "les points de la diagonalEdge, mais c'est pas à proprement parler "le bord" du plateau, c'est sa hauteur droite, comment nommer ?

Reply:
getBoardIsoPoints? getBoardThicknessDecorationPoints? J'sais pas trop, ce qui décrit au mieux ce que c'est !

Reply:
getBoardThicknessDecorationDiagonalPoints it is, je trouve qu'en l'absence du mot diagonal c'est mensongé, un peu ?

Reply:
techniquement une diagonale c'est un segment qui joint deux sommets d'un polygone, donc ça serait plutôt mensonger d'appeler un polygone / une arrête du polygone « diagonale » non ? tpr !

Reply:
boardsBottomEdge and boardsRightEdge pourraient le faire en fait là du coup non ?

Reply:
ouais, ou juste bottomEdge et rightEdge
Aussi, ça me perturbe visuellement, j'ai l'impression que le bottom edge il gagnerait à être un poil moins épais !

Reply:
Moi j'ai l'impression inverse !
Voilà, getRightEdge